openscap/xml/article_openscap.xml

```diff
@@ -33,6 +34,62 @@
         &openscap; and the <literal>&ssg;</literal>.
       </para>
     </abstract>
+    <meta name="title" its:translate="yes">Hardening &productname; with &openscap;</meta>
+    <meta name="series" its:translate="no">Products &amp; Solutions</meta>
+    <meta name="description" its:translate="yes">How to audit and harden &productname; with &openscap; and the &ssg;</meta>
+    <meta name="social-descr" its:translate="yes">Harden &productnameshort; with &openscap;</meta>
+    <meta name="task" its:translate="yes">
+      <phrase>Auditing</phrase>
+      <phrase>Compliance</phrase>
+    </meta>
+    <revhistory xml:id="rh-openscap">
```

Comment:
@sounix000 : Please check if the revhistory makes sense.

Reply:
Yes, the revision history is correct. Must have been a painfully boring task for you to dig through the commit and merge messages. Thanks for doing it; I will keep this in mind when I do PCI-DSS and HIPPA.